r-seurat-scripts-post-install-tests.bats

```diff
@@ -59,10 +59,35 @@
     
     [ "$status" -eq 0 ]
     [ -f  "$normalised_seurat_object" ]
+	}	
+
+
+
+ # run find transfer anchor with normalised_seurat_object as query and reference
+ 
+@test "Find transfer anchors" {
+   #if [ "$use_existing_outputs" = 'true' ] && [ -f "$normalised_seurat_object" ]; then
+     #   skip "$normalised_seurat_object exists and use_existing_outputs is set to 'true'"
+   # fi
+   
+    run seurat-find-transfer-anchor.R $normalised_seurat_object $normalised_seurat_object
```

Comment:
You'll also need to define a new output object for this step- you can do that by defining the variable about here: https://github.com/ebi-gene-expression-group/r-seurat-scripts/blob/edfced8f9167d1a9cec1564a62ced45ff71f6efa/r-seurat-scripts-post-install-tests.sh#L88, which you can then use in this bats step.